lib/jekyll-feed/feed.xml

```diff
@@ -11,20 +11,20 @@
 
   {% assign title = site.title | default: site.name %}
   {% if page.collection != "posts" %}
-    {% assign collection = page.collection | capitalize %}
-    {% assign title = title | append: " | " | append: collection %}
+    {% assign title = page.title | default: site.name %}
   {% endif %}
   {% if page.category %}
     {% assign category = page.category | capitalize %}
-    {% assign title = title | append: " | " | append: category %}
+    {% assign title = page.title | append: " | " | append: category %}
```

Comment:
This changes the behavior so that if `title` was set previously, now it's overwritten. Is that the intended behavior in all cases? It will do so for the `posts` collection, for example.

Reply:
I added a test to ensure that that `posts` collection retains its proper `<title>`, which is the site title.

Let me know if I have misunderstood your comment.